S21/myadlapalli/Language.py

```diff
@@ -0,0 +1,72 @@
+NO_CHANGE_LANGS = ['zh', 'ja', 'th']
```

Comment:
We can declare these variables inside the function for better reliability, otherwise, they can cause harm if not taken into consideration like its value changes without knowing